runtime/compiler/control/JitDump.cpp

```diff
@@ -483,9 +483,86 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
 
    try
       {
-      // if our compinfo is null, we are an application thread
+      // Process user defined options first. The user can specify any method signature to be recompiled at an arbitrary
+      // dump event. This is typically used for debugging purposes so we process this option first if it exists.
+      if (agent->dumpOptions != NULL)
+         {
+         char* indexOfDot = strchr(agent->dumpOptions, '.');
+         char* indexOfParen = strchr(agent->dumpOptions, '(');
+
+         if (indexOfDot != NULL && indexOfParen != NULL)
+            {
+            TR::VMAccessCriticalSection findUserMethod(TR_J9VMBase::get(jitConfig, crashedThread));
+
+            U_8* className = reinterpret_cast<U_8*>(agent->dumpOptions);
+            U_16 classNameLength = indexOfDot - agent->dumpOptions;
+            U_8* methodName = reinterpret_cast<U_8*>(indexOfDot + 1);
+            U_16 methodNameLength = indexOfParen - indexOfDot - 1;
+            U_8* methodSig = reinterpret_cast<U_8*>(indexOfParen);
+            U_16 methodSigLength = strlen(indexOfParen);
```

Comment:
nit: the `*` in these declarations binds more tightly to the variables than the target type so the space should be before `*`, not after:
```
U_8 *className = ...
```
(like line 511)

Reply:
Fixed in 2b02a6f

Reply:
None of the declarations above appear to have been 'fixed'.

Reply:
Apologies, I had fixed it in the VM portions of the implementation but not the JIT. The JIT is a bit of a Wild Wild West when it comes to formatting. I've made all the changes in 70f787d.